tests/types/array.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('StrongArray', () => {
 
 		it('should return fallback default when value is undefined', () => {
 			const sampleFallback = [7, 'koala'];
-			const result = makeArray(sampleFallback);
+			const result = makeArray(sampleFallback, undefined);
```

Comment:
Prefer to omit `undefined` arguments rather than use the value `undefined`. Example: `makeArray(sampleFallback, undefined` becomes `makeArray(sampleFallback)`.

- [ ] Removed `undefined` literal value from function call.